docs/sources/flow/reference/components/loki.write.md

```diff
@@ -115,6 +117,26 @@ based on a hash of the endpoint settings.
 
 {{< docs/shared lookup="flow/reference/components/tls-config-block.md" source="agent" >}}
 
+### wal block
```

Comment:
should we add a note here mentioning this is experimental for the time being?

Reply:
Yes, mark the header as experimental:

```suggestion
### wal block (experimental)
```